docs/content/docs/connectors/table/elasticsearch.md

```diff
@@ -239,7 +239,7 @@ Features
 ### Key Handling
 
 Elasticsearch sink can work in either upsert mode or append mode, it depends on whether primary key is defined.
-If primary key is defined, Elasticsearch sink works in upsert mode which can consume queries containing UPDATE/DELETE messages.
+If primary key is defined, Elasticsearch sink works in upsert mode which can consume queries containing UPDATE/INSERT messages.
```

Comment:
I don't think it's an INSERT :)

Please also make sure that you're follow the Code Contribution Guide, including the naming conventions for commits. See https://flink.apache.org/contributing/code-style-and-quality-pull-requests.html#commit-naming-conventions